notifications/views.py

```diff
@@ -1,19 +1,41 @@
+import logging
+import profile
+from django.forms import ValidationError
 from django_filters.rest_framework import DjangoFilterBackend
+from django.contrib.auth import get_user_model
 
 from rest_framework import generics, filters, status
 from rest_framework.pagination import PageNumberPagination
 from rest_framework.response import Response
 from rest_framework.permissions import IsAuthenticated
+from rest_framework.exceptions import NotFound
 
 from drf_yasg.utils import swagger_auto_schema
 from drf_yasg import openapi
 
-from .models import Notification, NotificationStatus
-from .serializers import NotificationSerializer, ExtendedNotificationSerializer
+from investors.models import InvestorProfile
+from startups.models import StartUpProfile
+from users.models import Role
+from .models import (
+    Notification,
+    NotificationStatus,
+    NotificationPreferences,
+    RolesNotifications
+)
+from .serializers import (
+    NotificationSerializer,
+    ExtendedNotificationSerializer,
+    NotificationPreferencesSerializer,
+    RolesNotificationsSerializer,
+    NotificationSerializerPost
+)
 from .filters import NotificationFilter
 
 
-class NotificationListView(generics.ListAPIView):
+User = get_user_model()
+logger = logging.getLogger('__name__')
+
```

Comment:
Same here. You call the logger, but there is no further access to it. If you need to implement logging in this file, then you should turn to the "django" logger, and also add logging to all operations at various levels.